topics/single-cell/tutorials/scrna-case_alevin/tutorial.md

```diff
@@ -76,10 +60,21 @@ recordings:
   youtube_id: 3ytm2AU6QUc
   speakers:
   - nomadscientist
+- youtube_id: BQriffof8yM
+  length: 49M
+  galaxy_version: 24.1.2.dev0
+  date: '2024-09-12'
+  speakers:
+  - MarisaJL
+  captioners:
```

Comment:
@pavanvidem: Do you want to be added to the captions? As I think you also worked on them.